src/js/__tests__/default-props-test.js

```diff
@@ -16,13 +16,15 @@ test('default theme is used', () => {
   const component = renderer.create(<Box background="brand" />);
   const tree = component.toJSON();
   expect(tree).toMatchSnapshot();
+  component.unmount();
```

Comment:
Can we avoid the repetitive `component.unmount();` and leverage a certain clean-up method on the test suite level after each test ends?

Reply:
Within the testing suites there is a combination of tests that use `render` and tests that use `renderer.create` so unmounting the component on all the tests would cause an error in tests that use `render`

Reply:
Saw this comment https://github.com/grommet/grommet/pull/4991#discussion_r582149917